configs/multilabel_classification/mobilenetv3_large.yml

```diff
@@ -65,7 +65,7 @@ train:
   nbd: True
   max_epoch: 80
   weight_decay: 5e-4
-  batch_size: 64
+  batch_size: 8
```

Comment:
Seems like 8 is not suitable for more or less large datasets.

Reply:
Yep, I forgot to return the batch size back